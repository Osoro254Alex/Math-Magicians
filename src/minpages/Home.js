import React from 'react';
import styles from './Home.module.css';

class Home extends React.PureComponent {
  render() {
    return (
      <div className={styles.home}>
        <h2 className={styles.head}>
          Welcome to our page!
        </h2>
        <p>
          Calculator is important do the complex calculation and save time to solve
          them. Calculator helps us to get answer fast with blink of eye for many
          complex and lengthy mathematical problems . hence its save our time , labor
          and reduce the waste of papers. It saves time and omits the chances of error
          and makes the complex calculations easy
        </p>
      </div>
    );
  }
}

export default Home;
