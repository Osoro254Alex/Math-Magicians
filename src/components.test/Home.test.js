import React from 'react';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

afterEach(cleanup);

describe('render welcome component without error', () => {
  test('welcome should render', () => {
    render(<Home />);
  });
  test('matches snapshot for new Header', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
