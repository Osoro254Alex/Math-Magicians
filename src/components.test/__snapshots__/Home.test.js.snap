// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render welcome component without error matches snapshot for new Header 1`] = `
<div
  className="home"
>
  <h2
    className="head"
  >
    Welcome to our page!
  </h2>
  <p>
    Calculator is important do the complex calculation and save time to solve them. Calculator helps us to get answer fast with blink of eye for many complex and lengthy mathematical problems . hence its save our time , labor and reduce the waste of papers. It saves time and omits the chances of error and makes the complex calculations easy
  </p>
</div>
`;
